Makefile

```diff
@@ -668,6 +668,11 @@ test-postgres-docker:
 	done
 .PHONY: test-postgres-docker
 
+# Make sure to keep this in sync with test-go-race from .github/workflows/ci.yaml.
+test-race: test-clean
```

Comment:
Why do we wipe the test cache when we run via Makefile? Adding `-count=1` should work the same and wouldn't trash the cache (e.g. for non-race tests).

Reply:
Should we update existing invocations to use `-count=1` as well?

Reply:
Actually, I'll make a separate PR for that and we can debate in there.

Reply:
Good suggestion, I think that would be preferable 👍🏻